release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2716,6 +2716,73 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-11"]
+=== RHSA-2024:2068 - {product-title} 4.15.11 bug fix and security update
+
+Issued: 2024-05-02
+
+{product-title} release 4.15.11, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2068[RHSA-2024:2068] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2071[RHSA-2024:2071] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.11 --pullspecs
+----
+
+[id="ocp-4-15-11-known-issues"]
+==== Known issue
+
+* The installation of {product-title} might fail when a performance profile is present in the extra manifests folder and targets the primary or worker pools. This is caused by the internal installation that processes the performance profile before the default primary and worker `MachineConfigPools` are created. As a workaround, include a copy of the stock primary or worker `MachineConfigPools` in the extra manifests folder. (link:https://issues.redhat.com/browse/OCPBUGS-27948[*OCPBUGS-27948*])
+
+[id="ocp-4-15-11-bug-fixes"]
+==== Bug fixes
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. This mean the `Port_Security` parameter was wrong. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected (link:https://issues.redhat.com/browse/OCPBUGS-30857[*OCPBUGS-30857*])
+
+* Previously, the image registry did not support AWS region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-31641[*OCPBUGS-31641*])
```

Comment:
```suggestion
* Previously, the image registry did not support Amazon Web Services (AWS) region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-31641[*OCPBUGS-31641*])
```
Probably well known enough that this isn't necessary but it looks weird bundled in with the others bugs that have it.